torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4916,6 +4922,9 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            autodiff_nonfusible_nodes=['aten::add', 'aten::mm'],
            sample_inputs_func=partial(sample_inputs_addmm, alpha=1, beta=1)),
     OpInfo('addmv',
+           # dtypes of beta and alpha should not affect the output type, hence the type conversion
+           ref=lambda M, mat, vec, beta, alpha: np.add(np.multiply(np.asarray(beta, dtype=M.dtype), M),
+           np.multiply(np.asarray(alpha, dtype=mat.dtype), np.dot(mat, vec))),
```

Comment:
`addmv` does matrix-vector multiplication (hence `np.dot`) while `addmm` does matrix-matrix multiplication (hence `np.matmul`) though `np.matmul` will work for both functions but it's not the best practice.

See 2nd point in: https://numpy.org/doc/stable/reference/generated/numpy.dot.html.

> If both a and b are 2-D arrays, it is matrix multiplication, but using matmul or a @ b is preferred.